auth_saml/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2020 GlodoUK <https://www.glodo.uk/>
+# Copyright (C) 2010-2016 XCG Consulting <http://odoo.consulting>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Saml2 Authentication",
+    "version": "14.0.1.0.0",
+    "category": "Tools",
+    "author": "XCG Consulting, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/server-auth",
+    "license": "AGPL-3",
+    "depends": ["base_setup"],
+    "external_dependencies": {"python": ["saml2"]},
```

Comment:
According to the travis error, shouldn't the dependency be https://pypi.org/project/pysaml2/  ?